### PR TITLE
feature/mx-1408 Fix public api tests

### PR DESCRIPTION
**Pull request by cutoffthetop**
- fix public api tests
  - some are dependant on the correct Dev Tenant configuration
  - those are being skipped in case of errors
- small fix to http base connector and wikidata
- bump dependency versions

### CODE REVIEW

File: mex/common/public_api/connector.py [hunk 1]
```diff
@@ -40,26 +43,31 @@ class PublicApiConnector(HTTPConnector):  # pragma: no cover
     TIMEOUT = 10
     API_VERSION = "v0"
 
-    def __init__(self, settings: BaseSettings) -> None:
-        """Create a new Pulic API connection.
+    token_provider: str
+    token_payload: SecretStr
```
Comment (carolinscholl) [inline mex/common/public_api/connector.py:47]: are these class variables ever used or set?

File: tests/public_api/test_connector.py [hunk 2]
```diff
@@ -137,7 +138,12 @@ def test_search_model_that_does_not_exist() -> None:
     random_id = Identifier.generate()
     connector = PublicApiConnector.get()
 
-    result = connector.search_model(ExtractedActivity, random_id)
+    try:
+        result = connector.search_model(ExtractedActivity, random_id)
+    except HTTPError as error:
+        if error.response.json().get("message") == "could not create Solr query":
+            pytest.skip("integration test failed due to misconfiguration")
```
Comment (carolinscholl) [inline tests/public_api/test_connector.py:145]: You're saying that the tests depend on the correct config and fail if the config is wrong. Now if the tests fail/are skipped, it is not directly clear if they are skipped due to wrong config or other problems, right? Is that intended? Or should the correct config be enforced for the tests?

File: mex/common/public_api/connector.py (hunk 1 above)
Comment (cutoffthetop) [reply mex/common/public_api/connector.py]: nope, good catch. fixed in bc430df

**Review by carolinscholl — APPROVED** [review]

**Review by carolinscholl — APPROVED** [review]

File: tests/public_api/test_connector.py (hunk 2 above)
Comment (cutoffthetop) [reply tests/public_api/test_connector.py:145]: to ensure correct field config we'd need to launch the entire "public API" in a github pipeline. the "could not create solr query" message is a strong indicator for a wrong/outdated config, that's why i put that in the if-clause. but you were right, i forgot to re-raise the http error if that message was not in the public-backend's response.

fixed in 218d82a

**Pull request merged**